fusor-ember-cli/app/controllers/review/installation.js

```diff
@@ -90,7 +90,7 @@ export default Ember.Controller.extend({
       // name is fqdn for managed host
       return (this.get("selectedRhevEngine.name"));
     }
-  }.property('selectedRhevEngine', 'engineDomain'),
+  }.property('selectedRhevEngine', 'selectedRhevEngine.name', 'engineDomain'),
```

Comment:
@cfchase, `property('selectedRhevEngine.name', 'engineDomain')` should work, just adding `.name` to the dependent attributes.